src/main/java/io/vertx/httpproxy/cache/CacheOptions.java

```diff
@@ -14,8 +21,12 @@
 public class CacheOptions {
 
   public static final int DEFAULT_MAX_SIZE = 1000;
+  public static final String DEFAULT_NAME = "__vertx.DEFAULT";
```

Comment:
Can you please add Javadoc for the default values, the constructors and the methods of this options class?

Similar to what we have for `HttpClientOptions`, for example.